eleventy.config.js

```diff
@@ -1,24 +1,39 @@
 const path = require('path');
+
 const fs = require('fs-extra');
 const { DateTime } = require('luxon');
-
+const createDOMPurify = require('dompurify');
+const { JSDOM } = require('jsdom');
 const syntaxHighlight = require('@11ty/eleventy-plugin-syntaxhighlight');
 const xmlFiltersPlugin = require('eleventy-xml-plugin');
+const Nunjucks = require('nunjucks');
 
 const slugify = require('./libs/slugify');
-const nunjucksEnvironment = require('./libs/templates');
 
 const inputDir = path.relative(__dirname, 'src/content');
 const wpInputDir = path.relative(__dirname, 'src/wp-content');
 const outputDir = path.relative(__dirname, 'build');
 
 const buildWordpressTheme = process.env.BUILD_WORDPRESS_THEME === '1';
 
+const window = new JSDOM('').window;
+const DOMPurify = createDOMPurify(window);
+const defaultNunjucksEnv = new Nunjucks.Environment();
+
 module.exports = function (eleventyConfig) {
   // Tell the config to not use gitignore for ignores.
   eleventyConfig.setUseGitIgnore(false);
 
-  eleventyConfig.setLibrary('njk', nunjucksEnvironment);
```

Comment:
Is this not needed to use a custom instance? Since there'll be a default one provided by 11ty.

Reply:
I think this definitely needs tests. Because we want to be sure that sanitization is definitely happening.

Reply:
After talking to Will he explained that the env created here is _only_ being used to get the safe filter to provide an enhanced version. It's not used as a custom Nunjucks env.